src/processor/operator/scan/scan_multi_rel_tables.cpp

```diff
@@ -90,7 +97,7 @@ bool ScanMultiRelTable::getNextTuplesInternal(ExecutionContext* context) {
         }
         const auto currentIdx = boundNodeIDVector->state->getSelVector()[0];
         if (boundNodeIDVector->isNull(currentIdx)) {
-            outState->getSelVectorUnsafe().setSelSize(0);
+            currentScanner = nullptr;
```

Comment:
We should cover this line. Can u open an issue.

Reply:
See https://github.com/kuzudb/kuzu/issues/4267. I believe we might have more cases not covered yet. Will address the issue later.